### ISSUE TITLE
Build the WNP fallback using Fluent

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#8196]
## Description

The fallback WNP needs to be built using Fluent for localized content

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Page is live and fully localized using Fluent for Tier 1 locales.
- [ ] Non Tier 1 locales see the existing default /whatsnew page (mobile promo).

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] Full Fluent L10n workflow is not yet in a finished state.

#### Tasks
- [x] Create Fluent file with final strings.
- [x] Create template to use said strings.
- [ ] Configure the file to be localized for Tier 1 locales.

**Comment by alexgibson** [comment]
@ejregithub do we have designs and copy for this fallback page?

**Comment by alexgibson** [comment]
I found design direction here: https://github.com/mozilla/bedrock/issues/8147

**Comment by alexgibson** [comment]
@sghoseWI seeing as this template isn't going to target any set URL or version number attached to it, can you please advise on what makes sense for the following analytics params?

- entrypoint
- utm_campaign

Thanks

**Comment by alexgibson** [comment]
This is now up on demo https://www-demo1.allizom.org/en-US/firefox/whatsnew/all/

Things still outstanding:

- Meta description string
- Analytics requirements

Blocked on:

- Fully working Fluent L10n workflow.

**Comment by sghoseWI** [comment]
@alexgibson I think the params you went with make sense for this.

**Comment by alexgibson** [comment]
Based on the discussion inyesterday's fluent/l10n status meeting, we agreed that we're unblocked on moving forward with reviewing & merging this PR.

The plan is to let this page live in production, showing in English only, until we get the L10n pipeline working. At that point we can file a separate issue for locale configuration and string extraction.
